### PR TITLE
docs(readme): add type to description

### PR DESCRIPTION
**Pull request by tmcgee123**
Also, updated the copyright to this year.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -445,12 +445,12 @@ app.controller('AppCtrl', ['$scope', function ($scope) {
 
 <h2 id="browser-compatibility">Browser Compatibility</h2>
 
-IE8 is not supported because I am utilising `Object.keys()`
+IE8 is not supported because I am utilizing `Object.keys()`
```
Comment (tymondesigns) [inline README.md]: I'm english :) https://en.wiktionary.org/wiki/utilise#Alternative_forms

File: README.md [hunk 2]
```diff
@@ -445,12 +445,12 @@ app.controller('AppCtrl', ['$scope', function ($scope) {
...
 
 To check if the browser natively supports local and session storage, you can do the following:
 
 ```js
-if (! locker.supported()) {
+if (!locker.supported()) {
```
Comment (tymondesigns) [inline README.md]: This is how I like my coding style

**Comment by tmcgee123** [comment]
I've reverted and squashed my changes. I apologize, JSCS does not run on markdown files and I was unaware of the styling you had set for unary operators.

**Comment by tymondesigns** [comment]
No worries, thanks for taking the time :+1:

**Pull request merged**